### ISSUE TITLE
Weird translation when applying slide filters

### ISSUE THREAD

**Issue by ArnaudAbreu** [ArnaudAbreu/PathAIA#31]
<img width="469" alt="Capture d’écran 2021-06-01 à 08 19 06" src="https://user-images.githubusercontent.com/35848886/120275740-122f7780-c2b2-11eb-9a05-b81fde9938f5.png">

I get misplaced patches when I apply a slide filter with `patchify_slide()` (see image above)

**Comment by schwobr** [comment]
The more I look into it, the more perplexed I am. Did you look into the csvs to check patch positions and see if the visu function makes sense with it ? I didn't find any obvious error in the code and I don't understand how slide filters could add this offset.

**Comment by ArnaudAbreu** [comment]
Yes I don't understand neither... I did not try to check what happens with the visu function, but I'm quite sure it comes from this function because all my extracted patches are relevant... It's not very important to solve it now, but I keep the issue open
